src/main/java/nl/tudelft/dnainator/ui/controllers/WindowController.java

```diff
@@ -23,53 +26,58 @@
  * </p>
  */
 public class WindowController {
-	private static final int EXT_LENGTH = 11; // .node.graph
-	private static final String EDGE = ".edge.graph";
 	@FXML private BorderPane root;
 	@FXML private View view;
-	private FileLoadService loadService;
+	private GraphLoadService graphLoadService;
+	private NewickLoadService newickLoadService;
 	private ProgressDialog progressDialog;
 
 	/**
 	 * Constructs a WindowController object, creating
-	 * a {@link FileLoadService} to go with it.
+	 * a {@link GraphLoadService} to go with it.
 	 */
 	@FXML
 	private void initialize() {
-		loadService = new FileLoadService();
+		graphLoadService = new GraphLoadService();
+		newickLoadService = new NewickLoadService();
```

Comment:
Shouldn't an ExceptionDialog be thrown when it fails loading in the newick file?